kuma/settings/common.py

```diff
@@ -1025,22 +1024,10 @@ def pipeline_one_scss(slug, **kwargs):
     },
     'payments': {
         'source_filenames': (
-            'js/components/payments/focus-visible.js',
-            'js/components/payments/payments-handler-utils.js',
-            'js/components/payments/payments-handler.js',
-            'js/components/payments/payments-faq.js',
-            'js/tooltip.js',
             'js/components/payments/payments-manage.js',
         ),
```

Comment:
There's now just this one js file in the payments bundle, but the bundle still gets included on every document page (and the homepage, I assume). Maybe you can move it so that it is only included in the /payments/recurring/management page.